django_pgviews/apps.py

```diff
@@ -24,6 +22,7 @@ def sync_pgviews(self, sender, app_config, **kwargs):
         
         if self.counter == total:
             log.info('All applications have migrated, time to sync')
+            from .models import ViewSyncer
```

Comment:
Add a comment here to just say we're preventing circular imports (or whatever the reason is for this).